python/cookiecutter-django.md

```diff
@@ -0,0 +1,538 @@
+# How starting a new Django project with Cookiecutter
+
+## Install Cookiecutter
```

Comment:
Here I would explain very shortly what does cookiecutter do. And also a link to cookiecutter